app/helpers/smart_proxies_helper.rb

```diff
@@ -65,4 +65,13 @@ def tabbed_features(proxy)
   def show_feature_version(feature)
     render :partial => 'smart_proxies/plugins/plugin_version', :locals => { :feature => feature }
   end
+
+  def subnet_label subnet
```

Comment:
Parenthesis missing to wrap the method arguments (thought rubocop would enforce this one)